packages/sockjs-shim/client.js

```diff
@@ -0,0 +1,9 @@
+// When the server.js module does not inject a <script> tag into the
+// <head> of the document, we still need a fake global shim for SockJS.
```

Comment:
Interesting, do you think this is better than some other part of the code checking for websockets/sockjs?

Reply:
Oh, that's a good point!